### DISCUSSION TITLE
Does the jupyter integration work with Google Colab?

### DISCUSSION THREAD

**Discussion by timlrx** [cylynx/motif.gl#165] [Q&A] [answered]
Google Colab would make it really convenient to share notebooks. Does pymotif work with it?

**Comment by jasonho-lynx** [comment, marked as answer]
Not at the moment, as Google Colab currently does not support custom widgets. You will be able to install `pymotif` on Colab, but when plotting it, you'll see an error along the lines of `Widget module ... not supported` in the browser console.

However, we've made a request in the main issue [here](https://github.com/googlecolab/colabtools/issues/498), so stay tuned, this might change!
